lms/static/scripts/frontend_apps/components/BasicLtiLaunchApp.js

```diff
@@ -179,9 +184,31 @@ export default function BasicLtiLaunchApp() {
   }, [authToken, authUrl, fetchContentUrl, lmsName]);
 
   if (ltiLaunchState.state === 'fetched-url') {
-    return (
-      <iframe width="100%" height="100%" src={ltiLaunchState.contentUrl} />
+    const iFrame = (
+      <iframe
+        key={sideBarKey}
+        width="100%"
+        height="100%"
+        className="js-via-iframe"
+        src={ltiLaunchState.contentUrl}
+      />
     );
+
+    if (lmsGrader) {
+      // Use the LMS Grader
+      return (
+        <LMSGrader
+          onChangeSelectedUser={username => {
```

Comment:
So this callback is only intended to alert this component to rebuild the iframe.  This is a change that is propagated from 2 components down, but I don't think it's enough of a use case to include a redux store as that would be more complex than a prop callback. We may rethink this when we get the submit feature working